client/src/pages/SinglePost.jsx

```diff
@@ -1,33 +1,70 @@
-import React from 'react'
-import Edit from "../img/temp2.jpg"
-import Delete from "../img/temp3.jpg"
-import Menu from "../components/Menu"
-import { Link } from 'react-router-dom'
+import React, { useEffect, useState } from "react";
+import Edit from "../img/temp2.jpg";
+import Delete from "../img/temp3.jpg";
```

Comment:
Are these supposed to be buttons? If so, can we use actual HTML button elements?

Reply:
Not too sure. These were here when I started working on this file.